packaging/language/maven_artifact.py

```diff
@@ -81,6 +80,12 @@
         required: true
         default: present
         choices: [present,absent]
+    validate_certs:
+        description: If C(no), SSL certificates will not be validated. This should only be used on personally controlled sites using self-signed certificates.
```

Comment:
Changed to: If C(no), SSL certificates will not be validated. This should only be set to no when no other   option exists.